test/e2e_node/resource_usage_test.go

```diff
@@ -135,12 +135,13 @@ func runResourceUsageTest(f *framework.Framework, rc *ResourceCollector, testArg
 		// sleep for an interval here to measure steady data
 		sleepAfterCreatePods = 10 * time.Second
 	)
+	pods := newTestPods(testArg.podsNr, ImageRegistry[pauseImage], "test_pod")
 
 	rc.Start()
+	defer cleanupPerfTest(f, pods)
```

Comment:
> Add a comment explaining why the manual cleanup is needed.

@yujuhong I added comment at the function definition of 'cleanupPerfTest', is it OK?
Otherwise the same comment will be put in both density and resource usage.